<?xml version="1.0" encoding="UTF-8"?>
<testng-results skipped="0" failed="1" ignored="0" total="5" passed="4">
  <reporter-output>
  </reporter-output>
  <suite name="Suite" duration-ms="19027" started-at="2023-06-01T22:22:17 IST" finished-at="2023-06-01T22:22:36 IST">
    <groups>
    </groups>
    <test name="Test" duration-ms="19027" started-at="2023-06-01T22:22:17 IST" finished-at="2023-06-01T22:22:36 IST">
      <class name="api.tests.UserTests">
        <test-method status="PASS" signature="setupData()[pri:0, instance:api.tests.UserTests@4671e53b]" name="setupData" is-config="true" duration-ms="1312" started-at="2023-06-01T22:22:17 IST" finished-at="2023-06-01T22:22:19 IST">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- setupData -->
        <test-method status="PASS" signature="testPostuser()[pri:1, instance:api.tests.UserTests@4671e53b]" name="testPostuser" duration-ms="11455" started-at="2023-06-01T22:22:19 IST" finished-at="2023-06-01T22:22:30 IST">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- testPostuser -->
        <test-method status="FAIL" signature="testReaduser()[pri:2, instance:api.tests.UserTests@4671e53b]" name="testReaduser" duration-ms="1406" started-at="2023-06-01T22:22:30 IST" finished-at="2023-06-01T22:22:32 IST">
          <exception class="java.lang.AssertionError">
            <message>
              <![CDATA[expected [201] but found [200]]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.AssertionError: expected [201] but found [200]
at org.testng.Assert.fail(Assert.java:99)
at org.testng.Assert.failNotEquals(Assert.java:1037)
at org.testng.Assert.assertEqualsImpl(Assert.java:140)
at org.testng.Assert.assertEquals(Assert.java:122)
at org.testng.Assert.assertEquals(Assert.java:907)
at org.testng.Assert.assertEquals(Assert.java:917)
at api.tests.UserTests.testReaduser(UserTests.java:40)
at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.lang.reflect.Method.invoke(Method.java:498)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:133)
at org.testng.internal.TestInvoker.invokeMethod(TestInvoker.java:598)
at org.testng.internal.TestInvoker.invokeTestMethod(TestInvoker.java:173)
at org.testng.internal.MethodRunner.runInSequence(MethodRunner.java:46)
at org.testng.internal.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:824)
at org.testng.internal.TestInvoker.invokeTestMethods(TestInvoker.java:146)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:146)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:128)
at java.util.ArrayList.forEach(ArrayList.java:1259)
at org.testng.TestRunner.privateRun(TestRunner.java:794)
at org.testng.TestRunner.run(TestRunner.java:596)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:377)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:371)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:332)
at org.testng.SuiteRunner.run(SuiteRunner.java:276)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:53)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:96)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1212)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1134)
at org.testng.TestNG.runSuites(TestNG.java:1063)
at org.testng.TestNG.run(TestNG.java:1031)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.AssertionError -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- testReaduser -->
        <test-method status="PASS" signature="testUpdateuser()[pri:3, instance:api.tests.UserTests@4671e53b]" name="testUpdateuser" duration-ms="1576" started-at="2023-06-01T22:22:32 IST" finished-at="2023-06-01T22:22:33 IST">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- testUpdateuser -->
        <test-method status="PASS" signature="testReadupdateduser()[pri:4, instance:api.tests.UserTests@4671e53b]" name="testReadupdateduser" duration-ms="1541" started-at="2023-06-01T22:22:33 IST" finished-at="2023-06-01T22:22:35 IST">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- testReadupdateduser -->
        <test-method status="PASS" signature="deleteUser()[pri:5, instance:api.tests.UserTests@4671e53b]" name="deleteUser" duration-ms="1533" started-at="2023-06-01T22:22:35 IST" finished-at="2023-06-01T22:22:36 IST">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- deleteUser -->
      </class> <!-- api.tests.UserTests -->
    </test> <!-- Test -->
  </suite> <!-- Suite -->
</testng-results>
